client/components/localized-moment/test/index.js

```diff
@@ -58,7 +64,11 @@ const setLocaleAndWait = async ( languageSlug, store, ...providerWrappers ) => {
 	await Promise.all( providerWrappers.map( getMomentProviderLoadingPromise ) );
 };
 
-describe( 'withLocalizedMoment', () => {
+// Test with both variants of the Label component: wrapped in HOC and using hooks
+describe.each( [
+	[ 'withLocalizedMoment', LabelWithMomentHOC ],
+	[ 'useLocalizedMoment', LabelWithMomentHook ],
```

Comment:
clever!

Reply:
I actually had to fight with the `describe.each` API, didn't like it very much, but the test suite runs in both variants!

Reply:
⚔️